src/coreclr/vm/comdelegate.cpp

```diff
@@ -2147,12 +2135,12 @@ extern "C" PCODE QCALLTYPE Delegate_GetMulticastInvokeSlow(MethodTable* pDelegat
     QCALL_CONTRACT;
     _ASSERTE(pDelegateMT != NULL);
 
-    PCODE fptr = (PCODE)NULL;
+    PCODE pStub = (PCODE)NULL;
 
     BEGIN_QCALL;
 
     DelegateEEClass *delegateEEClass = (DelegateEEClass*)pDelegateMT->GetClass();
-    Stub *pStub = delegateEEClass->m_pMultiCastInvokeStub;
+    PCODE pStub = delegateEEClass->m_pMultiCastInvokeStub;
```

Comment:
There are two variables named `pStub` in different scopes introduced by the QCall scope. The outer one is shadowed thus never assigned and the method would always return null.

Nit: is there a better name? `pCode` is already taken.

Reply:
This was an oversight. There should be just one pStub variable in the method.